### DISCUSSION TITLE
[HxMessengerService] - Change color of AddInformation

### DISCUSSION THREAD

**Discussion by econner20** [havit/Havit.Blazor#972] [General]
Is it possible to override the blue color of the AddInformation() to green? It would be ideal to have AddSuccess() that is green

**Comment by hakenr** [comment]
The `.AddInformation()` method is just an extension method that wraps around the underlying `.AddMessage()` method.  
Using `.AddMessage()`, you can pass any custom message with the styling of your choice (and you can create your own `.AddInformation()` extension method for easier usage).
